### DISCUSSION TITLE
Is there a plan to support arm neon/sve/sve2?

### DISCUSSION THREAD

**Discussion by Ocean-Lua** [greg7mdp/parallel-hashmap#230] [Q&A]
The arm ecosystem is rich. Do we consider supporting arm?

**Comment by greg7mdp** [comment]
arm is supported. Using arm simd instructions doesn't seem to make a significant difference vs using the C++ portable version, so I have so far chosen not to add custom simd code for arm, but I guess this could be revisited.
